tests/vmi_multiqueue_test.go

```diff
@@ -55,36 +55,40 @@ var _ = Describe("[sig-compute]MultiQueue", func() {
 
 	Context("MultiQueue Behavior", func() {
 		var availableCPUs int
+		const numCpus int32 = 3
 
 		BeforeEach(func() {
 			availableCPUs = libnode.GetHighestCPUNumberAmongNodes(virtClient)
 		})
-		It("[test_id:4599]should be able to successfully boot fedora to the login prompt with networking mutiqueues enabled without being blocked by selinux", func() {
+
+		DescribeTable("should be able to successfully boot fedora to the login prompt with multi-queue without being blocked by selinux", func(interfaceModel string, expectedQueueCount int32) {
 			vmi := tests.NewRandomFedoraVMIWithGuestAgent()
-			var numCpus int32 = 3
 			Expect(numCpus).To(BeNumerically("<=", availableCPUs),
 				fmt.Sprintf("Testing environment only has nodes with %d CPUs available, but required are %d CPUs", availableCPUs, numCpus),
 			)
 			cpuReq := resource.MustParse(fmt.Sprintf("%d", numCpus))
 			vmi.Spec.Domain.Resources.Requests[k8sv1.ResourceCPU] = cpuReq
 			vmi.Spec.Domain.Devices.NetworkInterfaceMultiQueue = pointer.Bool(true)
-			vmi.Spec.Domain.Devices.Rng = &v1.Rng{}
+
+			vmi.Spec.Domain.Devices.Interfaces[0].Model = interfaceModel
 
 			By("Creating and starting the VMI")
-			vmi, err := virtClient.VirtualMachineInstance(util.NamespaceTestDefault).Create(vmi)
+			vmi, err = virtClient.VirtualMachineInstance(util.NamespaceTestDefault).Create(vmi)
 			Expect(err).ToNot(HaveOccurred())
 			vmi = tests.WaitForSuccessfulVMIStartWithTimeout(vmi, 360)
 
 			By("Checking if we can login")
 			Expect(console.LoginToFedora(vmi)).To(Succeed())
 
 			By("Checking QueueCount has the expected value")
-			Expect(vmi.Status.Interfaces[0].QueueCount).To(Equal(numCpus))
-		})
+			Expect(vmi.Status.Interfaces[0].QueueCount).To(Equal(expectedQueueCount))
+		},
+			Entry("[test_id:4599] with default virtio interface", "virtio", numCpus),
+			Entry("with e1000 interface", "e1000", int32(1)),
```

Comment:
nit: Is the compiler requires this conversion here?
Also, one uses a constant (`numCpus`) and this one is not. This is strange.

Reply:
I do not like the fact that we are wasting an e2e test for this check. Investing in an assertion makes sense, it is almost free, but having a dedicated test that is wasting around a minute for this is a waste of resources IMO.

As I noticed you received a comment to make it a separate test, I can only repeat my disagreement to it, nothing more.